app/tutorials/example-applications/src/main/java/org/eclipse/mosaic/app/tutorial/vehicle/PerceptionApp.java

```diff
@@ -72,10 +72,12 @@ private void enablePerceptionModule() {
         // filter adding noise to longitudinal and lateral
         PositionErrorModifier positionErrorModifier = new PositionErrorModifier(getRandom());
 
-        SimplePerceptionConfiguration perceptionModuleConfiguration = new SimplePerceptionConfiguration(
-                VIEWING_ANGLE, VIEWING_RANGE,
-                simpleOcclusionModifier, distanceModifier, positionErrorModifier
-        );
+        SimplePerceptionConfiguration perceptionModuleConfiguration =
+                new SimplePerceptionConfiguration.Builder(VIEWING_ANGLE, VIEWING_RANGE)
+                        .addModifier(simpleOcclusionModifier)
```

Comment:
`.addModifiers(simpleOcclusionModifier, distanceModifier, positionErrorModifier)` would be also nice (?)

Reply:
good point! Added method to builder and applied to all use cases where more than one modifier is used